### ISSUE TITLE
[0.31] Big Map issue

### ISSUE THREAD

**Issue by lionkor** [Starystars67/BeamNG-MP#430]
![image](https://github.com/BeamMP/BeamMP/assets/29932116/746ce60f-adbc-40d2-8ef1-d75ac29824a4)

**Comment by lionkor** [comment]
Seems to be broken for me only when nothing is spawned in

**Comment by lionkor** [comment]
![image](https://github.com/BeamMP/BeamMP/assets/29932116/b1e8827f-7044-46b3-8159-2f6b2b126452)

**Comment by Starystars67** [comment]
This issue seems to no longer happen however we are unsure as to why.

If you do try to open the bigmap without a vehicle (or not tabbed to one) then the map does not open correctly.
If you do have a vehicle then it works as expected.

Maybe a Game bug?

**Comment by StanleyDudek** [comment]
> This issue seems to no longer happen however we are unsure as to why.
> 
> If you do try to open the bigmap without a vehicle (or not tabbed to one) then the map does not open correctly.
> If you do have a vehicle then it works as expected.
> 
> Maybe a Game bug?

In a completely clean 0.31, I loaded into small island, deleted the default truck, pressed M for map, and had no issues... switched map to Industrial, deleted truck, M for map, no issue.

I don't think it's a game bug

**Comment by lionkor** [comment]
It works fine without MP

**Comment by A56781** [comment]
Can you make it work fine with MP

**Comment by A56781** [comment]
add fireexd on discord

**Comment by Starystars67** [comment]
Spot the problem xD
<img width="1790" alt="image" src="https://github.com/BeamMP/BeamMP/assets/15389482/8b3496c7-cb9f-4f82-b882-578c71e3bde1">

**Comment by Starystars67** [comment]
It looks like this issue occurs only when there are no vehicles spawned in the session (which BeamMP does this so you can spawn a vehicle of your choice once loaded).

This seems to be similar to the issue encountered in the level editor mentioned in this issue here: #436

**Comment by lionkor** [comment]
@Starystars67 @Olrosse what's the status here? i still see this happening

**Comment by lionkor** [comment]
Seems like this shows markers from a different map? Also saw someone say it happens when you switch from one server to another, with a different map.

**Comment by dennorske** [comment]
I can confirm this.

I self host a server and by simply logging out of my server (not close the game), change map (And restart the server) - rejoining the server it has the the map icon bug. The icons are from the previous map as you say @lionkor.

I tried changing from Jungle Rock island to West coast usa. 


EDIT: I also tried to see if it would clear if i went to freeroam, but, it just "stacks" up the icons from the previous two maps. So a full game restart is needed to get rid of it currently, AFAIK
